cob/cob.c

```diff
@@ -69,6 +69,12 @@ enum {
 	M0_COB_EA_MAX   = 4096
 };
 
+struct bc_entry {
+	struct m0_buf       be_key;
+	struct m0_buf       be_rec;
+	struct m0_list_link be_link;
```

Comment:
Use tlink here.

Reply:
I will remove this structure, as it is not used anymore now.

Reply:
Done.